src/argilla_sdk/settings/_question.py

```diff
@@ -398,9 +398,11 @@ def question_from_model(model: QuestionModel) -> QuestionType:
         raise ValueError(f"Unsupported question model type: {model.settings.type}")
 
 
-def _render_values_as_options(values: Union[List[str], List[int]]) -> List[Dict[str, str]]:
+def _render_values_as_options(values: Union[List[str], List[int], Dict[str, str]]) -> List[Dict[str, str]]:
     """Render values as options for the question so that the model conforms to the API schema"""
-    if isinstance(values, list) and all(isinstance(value, str) for value in values):
+    if isinstance(values, dict):
```

Comment:
Should we adapt the [reverse method](https://github.com/argilla-io/argilla-python/pull/192/files#diff-516606dd5805b66ae35d14be900b9ec9f18a9e456ada3c38a41f3591427bef92L412) to convert options into a dictionary since the value type supports this?